test/controllers/ConfirmationController.test.ts

```diff
@@ -22,11 +22,12 @@ describe('ConfirmationController', () => {
 
         const appeal = {
             penaltyIdentifier: {
-                companyNumber: '00345567',
+                companyNumber: 'NI000000',
```

Comment:
I thought, wouldn't it be easier to change company number that was authenticated to `00345567`. Just a thought.

Reply:
yep! probably would have been a few less changes